backend/capellacollab/core/authentication/api_key_cookie.py

```diff
@@ -15,29 +14,24 @@
 from capellacollab.core import database
 from capellacollab.events import crud as events_crud
 
-from . import exceptions, get_authentication_entrypoint
+from . import exceptions, keystore
 
 log = logging.getLogger(__name__)
-ep = get_authentication_entrypoint()
-ep_main = importlib.import_module(".__main__", ep.module)
 
 
-class JWTBearer(security.HTTPBearer):
+class JWTAPIKeyCookie(security.APIKeyCookie):
     def __init__(self, auto_error: bool = True):
-        super().__init__(auto_error=auto_error)
+        super().__init__(name="access_token", auto_error=auto_error)
 
-    async def __call__(  # type: ignore
-        self, request: fastapi.Request
-    ) -> str | None:
-        credentials: security.HTTPAuthorizationCredentials | None = (
-            await super().__call__(request)
-        )
+    async def __call__(self, request: fastapi.Request) -> str | None:
+        token: str | None = await super().__call__(request)
 
-        if not credentials or credentials.scheme != "Bearer":
+        if not token:
             if self.auto_error:
                 raise exceptions.UnauthenticatedError()
             return None
-        if token_decoded := self.validate_token(credentials.credentials):
+
+        if token_decoded := self.validate_token(token):
             self.initialize_user(token_decoded)
```

Comment:
True, I was always just using the admin that was initially created. I have moved the logic of creating a non-existing user to the `validate_id_token` function, so instead of throwing a `UserNotFoundError` we just create the user there. This also has the advantage that we can completely remove the user initialization from the `JWTAPIKeyCookie`, making this class much simpler.